resources/js/components/MapContainer.vue

```diff
@@ -1,21 +1,25 @@
 <template>
     <div>
         <div class="h-full" ref="mapEl"></div>
-        <MapPopup v-if="mapRef" :map="mapRef" :feature="popupFeature" />
+        <div class="hidden">
+            <div ref="popup">
+                <slot
+                    v-if="popupFeature"
+                    name="popup"
+                    :feature="popupFeature"
+                ></slot>
+            </div>
+        </div>
```

Comment:
This look suspicious would this work?

```suggestion
            <div ref="popup">
                <slot
                    v-if="popupFeature"
                    name="popup"
                    :feature="popupFeature"
                ></slot>
        </div>
```

Reply:
This needs to be `display: none` at some point so that it doesn't show up when there's no active artwork.

It could be handled right in the popup, but I think it fits here better because the hiding is an implementation detail of `MapContainer`.